pkg/rpc/rpc.proto

```diff
@@ -15,6 +15,9 @@ service Draft {
 	// GetLogs returns the logs for a particular draft build.
 	rpc GetLogs(GetLogsRequest) returns (GetLogsResponse) {}
 
+	// GetLatestBuildID returns the latest build ID for a given application
+	rpc GetLatestBuildID(GetLatestBuildIDRequest) returns (GetLatestBuildIDResponse) {}
```

Comment:
Can we instead rename the RPC call as fetching the latest build instead of just its ID? e.g.

```
// GetLatestBuild returns the latest build for a given application
rpc GetLatestBuild(GetLatestBuildIDRequest) returns (GetLatestBuildResponse) {}
```

Even if functionally it's the same, this way we don't have to introduce new RPC functions any time we introduce a new field to a Build

In this sense, the response would then look something like this:

```
message GetLatestBuildResponse {
	string ID = 1;
}
```

Does that make sense?